test-app/tests/integration/components/file-input-field-test.gts

```diff
@@ -300,7 +297,9 @@ module('Integration | Component | FileInputField', function (hooks) {
 
     // Verify files are there
     assert.dom('[data-files]').exists();
-    await triggerEvent('button', 'click');
+    await triggerEvent('[data-delete-file]', 'click');
+
+    
```

Comment:
Super nit: I think VSCode prettier may complain about this being two returns rather than 1, but may be okay?

Reply:
I removed it manually...tried opening up VS Code to format this, but then got red lines everywhere...